### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KaterynaEV**
- [DEMO LINK](https://KaterynaEV.github.io/layout_catalog/)
- [TEST REPORT LINK](https://KaterynaEV.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
-  two failed tests cannot find tag main. wrap div with class container in main tag
- make separate file for each  block.

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,478 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <!-- #endregion -->
```
Comment (OleziO) [inline src/index.html:13]: Remove all comments from the code.

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,478 @@
...
+          alt="Moyo logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
```
Comment (OleziO) [inline src/index.html:31]: Make links more meaningful.

Examples:

#apple, #samsung etc.

File: src/styles/blocks/stars.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      margin-left: 4px;
+    }
+  }
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(n) {
```
Comment (OleziO) [inline src/styles/blocks/stars.scss:21]:
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n + 5) {
```

Make all code the same style

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by OleziO — CHANGES_REQUESTED** [review]
Fix all comments

**Review by Esceype — APPROVED** [review]
Perfect!

**Pull request closed without merging**